apps/prairielearn/src/pages/studentAssessment/studentAssessment.js

```diff
@@ -179,31 +181,28 @@ router.post(
       );
       res.redirect(res.locals.urlPrefix + '/assessment_instance/' + assessment_instance_id);
     } else if (req.body.__action === 'join_group') {
-      await groupAssessmentHelper.joinGroup(
+      await joinGroup(
         req.body.join_code,
         res.locals.assessment.id,
         res.locals.user.user_id,
         res.locals.authn_user.user_id,
-      );
+      ).catch((err) => flash('error', err.message));
```

Comment:
I don't like this pattern; any unexpected application errors will be entirely invisible to Sentry and our logs.

Reply:
Handled as custom error class in 65872d4f3.